Gruntfile.js

```diff
@@ -41,13 +41,13 @@ module.exports = function (grunt) {
 			devStyles: {
 				expand: true,
 				cwd: 'src',
-				src: '**/widgets.css',
+				src: ['**/widgets.css', '**/example.css'],
```

Comment:
I think it's still required for the current dojo theme (it copies that `widgets.css` as well.) I only noticed this because of my testing, fwiw.